webdriver-spec.html

```diff
@@ -2438,12 +2438,12 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
 
         <table class=simple>
          <tr>
-          <th>Platform
-          <th>Operating System
+          <th>Key
+          <th>System
          </tr>
-         <tr><td>linux</td><td>Any OS based upon the Linux kernel and X Windows.</td></tr>
-         <tr><td>mac</td><td>Any version of Apple's OS X.</td></tr>
-         <tr><td>windows</td><td>Any version of Microsoft's Windows, including desktop and mobile versions.</td></tr>
+         <tr><td>"<code>linux</code>"</td> <td>Any system based upon the Linux kernel.</td></tr>
```

Comment:
Android doesn't fit into this category, but is based on the Linux kernel, which is why we had the qualifier about X Windows.

Reply:
Not all Linux systems use X.

Reply:
We need to differentiate Android from Linux. "Any server or desktop system based upon the linux kernel"?

Reply:
Makes sense, fixed.